src/store/middlewares/peers.js

```diff
@@ -10,7 +11,10 @@ const peersMiddleware = store => next => (action) => {
 
   switch (action.type) {
     case actionTypes.storeCreated:
-      store.dispatch(activePeerSet({ network }));
+      // It changes network back to mainnet when we want to autologin for explorer account
```

Comment:
The comment explains well why you added the if statement and makes sense in the context of this PR. But if I would read it after the PR is merged, I would expect that the comment explains why the thing inside the condition is executed, and then it doesn't make much sense.